### DISCUSSION TITLE
Bitrate Caping in Web and Mobile

### DISCUSSION THREAD

**Discussion by shubhankarMitra** [videojs/video.js#8838] [Q&A]
I want to cap the bitrate to : desktop max 2500kbps and mweb 1000kbps

How to do this in Video.js

**Comment by mister-ben** [comment]
You can use videojs-contrib-quality-levels for this. Similar to the example in the readme, but enable or disable based on the qualityLevel's `bitrate` rather than `height`.

https://github.com/videojs/videojs-contrib-quality-levels

**Reply by shubhankarMitra** [reply]
my `video.qualityLevels()` array comes empty.
Thats what has been bothering me how to populate that
